metricbeat/module/windows/services/_meta/fields.yml

```diff
@@ -0,0 +1,36 @@
+- name: services
+  type: group
+  description: >
+    `services` contains the status for windows services.
+  fields:
+    - name: uptime
+      type: group
+      description: >
+        `uptime` contains the service uptime metric.
+      fields:
+      - name: ms
+        type: long
+        format: duration
+        input_format: milliseconds
+        description: >
+          The service uptime in milliseconds.
+
+    - name: service_name
+      type: keyword
+      description: >
+        The service name.
+
+    - name: display_name
+      type: keyword
+      description: >
+        The display name of the service.
```

Comment:
For the fields that are enums it would be nice to document what the possible values are. I would simply add a sentence that says "The possible values are x, y, and z".

Reply:
Some special format for the values like `x`, `y` and `z`

Reply:
If you like 👍